Orange/widgets/unsupervised/owpca.py

```diff
@@ -46,8 +40,6 @@ class Outputs:
 
     ncomponents = settings.Setting(2)
     variance_covered = settings.Setting(100)
-    batch_size = settings.Setting(100)
-    address = settings.Setting('')
     auto_update = settings.Setting(True)
```

Comment:
`auto_update` needs to be removed too right?

Reply:
resolved